content/22.vaccines.md

```diff
@@ -405,6 +405,116 @@ Additionally, given that severe SARS-CoV-2 may be associated with a dysregulated
 The article [@doi:10.1016/j.cell.2020.04.042] proposes that trained immunity might lead to an earlier and stronger response, which could in turn reduce viremia and the risk of later, detrimental immunopathology.
 While trained immunity is an interesting possible avenue to complement vaccine development efforts through the use of an existing vaccine, additional research is required to assess whether the BCG vaccine is likely to confer trained immunity in the case of SARS-CoV-2.
 
+### Global Vaccine Status and Distribution
+The unprecedented development of COVID-19 vaccines in under a year since the beginning of the pandemic now requires rapid global vaccine production and distribution plans.
+The development of vaccines are costly and complicated, but vaccine distribution can be just as challenging.
+Logistical considerations such as transport, storage, equipment (syringes), the workforce to administer the vaccines, and a continual supply from the manufacturers to meet global demands all must be accounted for and will vary globally due to economic, geographic, and sociopolitical reasons [@doi:10.1016/j.eclinm.2020.100674; @doi:10.1126/science.abe2803; @doi:10.1126/sciadv.abf1374]. 
+Deciding on the prioritization and allocation of the COVID-19 vaccines is also a challenging task due to ethical and operational considerations.
+Various frameworks, models, and methods have been proposed to tackle these issues with many countries, regions or States as is the case in the U.S., devising their own distribution and administration plans  [@doi:10.1126/science.abe6959; @doi:10.1038/s41598-020-78447-3; @doi:10.1016/j.vaccine.2020.12.059; @doi:10.1136/medethics-2020-107036; @doi:10.1101/2021.01.12.21249694]. 
+The majority of the distribution plans prioritize health care workers, the elderly, the immunocompromised, and those most vulnerable due to comorbidities should be offered the vaccines first, before targeting the rest of the population who are less likely to have severe outcomes [@url:https://ourworldindata.org/covid-vaccination-policy].
+As of March 6th, 2021, approximately 319 million vaccine doses have been administered in at least 118 countries worldwide using 10 different vaccines [@url:https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html; @url:https://www.bloomberg.com/graphics/covid-vaccine-tracker-global-distribution/]. 
```

Comment:
That would be awesome if we could. It would certainly keep us up to date.


The New York Times, Bloomberg and a few other websites we cite all use maps to show vaccine approvals and distributions. I speak hypothetically because I know nothing about code only that I want to learn when I get time (which is never haha), but could you scrape data from them to make our our own maps like the ones for the clinical trials data?

Reply:
So it just depends on where they're pulling data from! When it's a CSV file (or a text file, or any structured file type) that's on the internet like in that link, it's pretty easy to bring the data into python and then make a map! But for something like Bloomberg, I would guess they are cagey about sharing their data (given that I usually can't see their articles!)

No worries about not having time to learn to code -- it's much easier to learn out of need rather than as something you have an abstract interest in! When the world opens back up, hopefully we'll be able to get [Software Carpentry](https://software-carpentry.org/) workshops running in Philly, where the goal is to offer a supportive space where scientists can dabble with learning to code!

Reply:
They may be getting their info from the world of data page we cite. They have a load of COVID-related data available. 

I definitely want to learn for myself, not being able to do even simple things in R or scrape data into a functional output is not a major issue right now, but would it would be very useful in the future.

Reply:
Excellent, I will keep you on my list of people to bug when/if I eventually get a workshop organized!

Reply:
Definitely do!!